pootle/apps/pootle_app/models/directory.py

```diff
@@ -80,6 +80,7 @@ class Meta(object):
         default_permissions = ()
         app_label = "pootle_app"
         index_together = ["obsolete", "pootle_path"]
+        unique_together = ["obsolete", "tp", "tp_path"]
```

Comment:
Can it be the same tp-tp_path combination be at the same time obsolete and not obsolete? If not I would drop `obsolete` from here.